pipes/WDL/workflows/tasks/ncbi.wdl

```diff
@@ -101,19 +111,25 @@ task prepare_genbank {
         ${authors_sbt} \
         ${sep=' ' assemblies_fasta} \
         . \
-        ${'--master_source_table=' + genbankSourceTable} \
-        ${'--sequencing_tech=' + sequencingTech} \
-        ${'--biosample_map=' + biosampleMap} \
-        ${'--coverage_table=' + coverage_table} \
-        ${'--comment=' + comment} \
+        --mol_type ${molType} \
+        --organism "${organism}" \
+        --biosample_map ${biosampleMap} \
+        --master_source_table ${genbankSourceTable} \
+        ${'--coverage_table ' + coverage_table} \
+        --comment "${comment}" \
+        --sequencing_tech "${sequencingTech}" \
         --loglevel DEBUG
-    tar -czpvf ${out_prefix}.tar.gz *.val *.cmt *.fsa *.gbf *.sqn *.src *.tbl
+    mv errorsummary.val errorsummary.val.txt # to keep it separate from the glob
   }
 
   output {
     Array[File] sequin_files = glob("*.sqn")
-    File        ncbi_package = "${out_prefix}.tar.gz"
-    File        errorSummary = "errorsummary.val"
+    Array[File] structured_comment_files = glob("*.cmt")
```

Comment:
Should some of these be specified as one-or-more `Array[File]+`? (Or does it matter for outputs?)

Reply:
I played wth that at one point. maybe `sequin_files` makes sense. The only reason it matters for WDL outputs is if a workflow connects the output of one stage to the input of the next, and the subsequent input has a multiplicity restriction on it (the compiler will enforce that the upstream output has a compatible multiplicity).